### PR TITLE
Apache Nifi package

### PR DESCRIPTION
**Pull request by us3r**
Hereby the first version of the newest version of Apache Nifi v1.1.1. With using ENV variable you can manage each parameter in nifi.properties file. 
Please keep in mind that cluster mode is working in "experimental" mode. Full support for that functionality will be added in next release.

### CODE REVIEW

**Comment by ryadav88** [comment]
@us3r thanks. Can I also ask you to write short getting started doc for this package so users don't have any issues installing and understanding how it works?

You can create a PR against https://github.com/dcos/examples and refer other examples listed there. These examples will finally go here: https://dcos.io/docs/1.8/usage/tutorials/. Please follow the template of other examples here.

This is also now part of the package submission guidelines: https://github.com/mesosphere/universe#submit-your-package

**Comment by us3r** [comment]
ok no problem ;) it is done.

File: repo/packages/N/nifi/0/package.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  "name": "nifi",
+  "version": "1.1.1-0.0.1",
+  "minDcosReleaseVersion" : "1.8",
+  "scm": "https://git-wip-us.apache.org/repos/asf?p=nifi.git",
+  "maintainer": "mariusz.derela@gmail.com",
+  "website": "https://nifi.apache.org",
+  "framework": false,
+  "description": "An easy to use, powerful, and reliable system to process and distribute data.",
+  "tags": ["nifi","dataflow","apache","flow","data"],
+  "preInstallNotes":"Nifi on DC/OS is in alpha and there may be bugs, incomplete features, incorrect documentation or other discrepancies.",
```
Comment (ryadav88) [inline repo/packages/N/nifi/0/package.json]: Replace the `preInstallNotes` with: ```This DC/OS Service is currently EXPERIMENTAL. There may be bugs, incomplete features, incorrect documentation, or other discrepancies. Experimental packages should never be used in production!```

File: repo/packages/N/nifi/0/package.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{
+  "packagingVersion": "3.0",
+  "name": "nifi",
+  "version": "1.1.1-0.0.1",
+  "minDcosReleaseVersion" : "1.8",
+  "scm": "https://git-wip-us.apache.org/repos/asf?p=nifi.git",
+  "maintainer": "mariusz.derela@gmail.com",
+  "website": "https://nifi.apache.org",
+  "framework": false,
+  "description": "An easy to use, powerful, and reliable system to process and distribute data.",
```
Comment (ryadav88) [inline repo/packages/N/nifi/0/package.json]: Please add a detailed description of what Nifi is.

Comment (ryadav88) [inline repo/packages/N/nifi/0/package.json]: Add a link to dcos/examples at the end of `description`. Something like: 
```Installation Documentation: https://github.com/dcos/examples/tree/master/1.8/nifi```

File: repo/packages/N/nifi/0/package.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  "tags": ["nifi","dataflow","apache","flow","data"],
+  "preInstallNotes":"Nifi on DC/OS is in alpha and there may be bugs, incomplete features, incorrect documentation or other discrepancies.",
+  "postInstallNotes": "Nifi has been successfully installed.\nDocumentation can be found at https://nifi.apache.org/docs.html\nPlease keep in mind that first start of nifi can take a more than 60 seconds, so please be patient.",
+  "postUninstallNotes": "Nifi has been uninstalled.",
+  "licenses": [
+    {
+      "name": "Apache License Version 2.0",
+      "url": "https://raw.githubusercontent.com/elastic/kibana/master/LICENSE.md"
```
Comment (ryadav88) [inline repo/packages/N/nifi/0/package.json]: replace with https://github.com/apache/nifi/blob/master/LICENSE

**Comment by us3r** [comment]
@ryadav88 that information with "/nifi" is ok - it is standard Nifi behaviour when you have a single nifi installation (not in the cluster mode). I've put that information to the POST installation info - that in case when you have a single node - you should type http://NIFI_ENDPOINT:PORT/nifi instead of http://NIFI_ENDPOINT:PORT

**Comment by ajit425** [comment]
When I tried to enable clustering, the service did not start. It is failing the health check and DC/OS keep creating new container and trying to start it again. Did you seen similar problems in your testing. Also, Any plan yo update the package to latest version of nifi 1.3.0

**Pull request merged**